### DISCUSSION TITLE
Is it possible to decorate collection? Not individual item

### DISCUSSION THREAD

**Discussion by denrolya** [api-platform/api-platform#2119] [Q&A]
Question is simple, using collection 'get' operation on a doctrine entity I want the result to include a custom field on a same level `hydra:member` is. Is it possible?

**Comment by SherinBloemendaal** [comment]
+1, would be very nice. I would like to return a key in the collection response like `custom:maxItems` so the frontend knows how many slots the user has according to their subscription (`50 slots of the 55 slots used` for example), and its different per entity.
